WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/BaseUsernameChangerFullScreenDialogFragment.java

```diff
@@ -209,6 +210,12 @@ public void onStop() {
         mDispatcher.unregister(this);
     }
 
+    @Override
+    public void onDestroy() {
+        mGetSuggestionsHandler.removeCallbacksAndMessages(null);
```

Comment:
This should help with the possible memory leak in the runnable passed to the handler.